content/basics/use-ably.textile

```diff
@@ -0,0 +1,139 @@
+* Your application needs to register its presence on a channel, or listen for others becoming present in real time.
+* You want to maintain a persistent connection to Ably, attach to one or more channels, and publish and subscribe to messages.
+* You want to publish messages at a very high volume, which is much more efficient over a realtime connection, as publishes are pipelined over a websocket.
+* You need to publish messages with very low latency: Realtime publishes have much lower overhead than REST publishes, so have significantly lower latency.
+* You're running into your hourly API request limits. REST publishesr use a separate API request per publish, so if you're hitting your limits there, this may be a reason to use the Realtime library.
+
+The Ably Realtime client library is available in the most popular languages with "a number of SDKS":https://ably.com/download available.
+
+h2(#rest-api). Ably REST API
+
+The Ably REST API provides a way for a wide range of server and client devices to communicate with the Ably service over REST. The REST API does not provide a realtime long-lived connection to Ably, but in all other respects is a simple subset of the full realtime messaging API.
+
+The primary use-case for the REST API is for servers that are part of the back-end of an application such as a web application, that publish messages, issue access tokens (temporary and client-specific) for its various clients, obtain message and presence history and retrieve statistics.
+
+Reasons to use the REST library:
+
+* Your application server is used to primarily issue tokens for clients and/or publish messages on channels.
+* Your application is mostly stateless i.e. you process a request or respond to an event, and then move onto the next request or event without any previous state carrying through.
+* You prefer a synchronous request over an asynchronous request.  Note not all REST libraries are synchronous, but where the platform offers a synchronous and asynchronous approach, the REST libraries are more often synchronous.
+
+The Ably REST client library is available in the most popular languages with "a number of SDKS":https://ably.com/download available.
+
+h2(#protocol-adapters). Protocol Adapters
+
+Although Ably Client Library SDKs are the recommended method for connecting to Ably, we also provide a number of protocol adapters as an alternative method for connecting to Ably. 
+
+The advantage to protocol adapters is that they require fewer resources in terms of memory and network overhead such as in smaller footprint devices, or on a platform where an Ably Client Library SDK isn’t available such as an Arduino-based IoT wearable. The potential drawback to consider when evaluating protocol adapters is that they do not support the full set of Ably features, for example the MQTT protocol adapter does not support presence, and the SSE protocol adapter does not support automatic token renewal.
```

Comment:
This is the sort of information we should be discussing on this page - to help users identify how they should be using / connecting to Ably. Please be wary of reusing the exact same content from the protocol adapters page though, and considering contrasting more with the SDKs.